test/api/units/feedback_test.rb

```diff
@@ -10,11 +10,10 @@ def app
   end
 
   def test_get_awaiting_feedback
-    random_unitrole = UnitRole.tutors.sample
-    unit = random_unitrole.unit
-    user = random_unitrole.user
+    user = User.first
+    unit = Unit.first
 
-    expected_response = unit.tasks_awaiting_feedback
+    expected_response = unit.tasks_awaiting_feedback(user)
```

Comment:
Tests were incomplete and are on another branch yet to be pushed, I'll remove these from here (weren't meant to be in PR)